docs/metric_deletion_retention.md

```diff
@@ -0,0 +1,47 @@
+# Metric Deletion 
+
+Promscale exposes an API for deletion of series. This works same as prometheus delete API.
+
+### Promscale Delete API
+
+URL query parameters:
+
+* **match[]=<series_selector>**: Repeated label matcher argument that selects the series to delete. At least one match[] argument must be provided.
+* **start=<rfc3339 | unix_timestamp>**: Start timestamp. Optional and defaults to minimum possible time.
+* **end=<rfc3339 | unix_timestamp>**: End timestamp. Optional and defaults to maximum possible time.
```

Comment:
@Harkishen-Singh the start and end options are not supported yet, correct? We should take them out of here then and put a note at the top of this section.

Reply:
Yes, they are not supported. We delete the complete series that match the matchers. (Provided in the `match[]` param)